melosys-eessi-app/src/main/resources/application-local.properties

```diff
@@ -28,4 +28,5 @@ KAFKA_SEDMOTTATT_TOPIC=eessi-basis-sedMottatt-v1
 KAFKA_SEDSENDT_TOPIC=eessi-basis-sedSendt-v1
 KAFKA_SEDHENDELSER_GROUPID=melosys-eessi-sedHendelser-local
 KAFKA_EESSIMELDING_TOPIC=privat-melosys-eessi-v1-local
+KAFKA_AIVEN_EESSIMELDING_TOPIC=teammelosys.eessi.v1-local
```

Comment:
Må vi ha et nytt topic for aiven?

Reply:
ja det må vi nok. Når topic opprettes vi aiven via en yaml fil så brukes namespace som prefix til topic. Dessuten er de ikke private lenger

https://github.com/navikt/melosys-iac